packages/create-cloudflare/templates/scheduled/ts/package.json

```diff
@@ -4,8 +4,8 @@
 	"private": true,
 	"scripts": {
 		"deploy": "wrangler deploy",
-		"dev": "wrangler dev",
-		"start": "wrangler dev",
+		"dev": "wrangler dev --test-scheduled",
+		"start": "wrangler dev --test-scheduled",
```

Comment:
Should these changes be made in the `js` directory too?

Reply:
Yeah, go for it 👍